CHANGELOG.md

```diff
@@ -12,6 +12,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * SwiftLint Strict Mode on GitHub Actions
 * PayOne SEPA payment #APPS-513
 
+### Changed
+* Reset IBAN number on SEPA payment remove action #APPS-590
+
```

Comment:
Sorry durch den Release heute morgen ist es nun noch in der falschen Kategorie, da es erst in Version 0.26.0 kommt 😢

Reply:
e319303fb6127b656a3bd36258309d63fe641045